### PR TITLE
Version 3.15.0 of spotless

### PR DESCRIPTION
**Pull request by wslulciuc**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -2,7 +2,7 @@ plugins {
   id 'java'
   id 'application'
   id 'com.github.johnrengelman.shadow' version '2.0.4'
-  id "com.diffplug.gradle.spotless" version '3.14.0'
+  id "com.diffplug.gradle.spotless" version '3.15.0'
```
Comment (sshah-wework) [inline build.gradle]: Seems super weird that the old version went away -- should it be in maven? wondering if we can ask gradle to retry pulling dependencies a couple of times before failing entirely

Comment (sshah-wework) [reply build.gradle]: So here's 3.14 in the gradle repo: https://plugins.gradle.org/plugin/com.diffplug.gradle.spotless/3.14.0

Technically we should not have had to do a version bump here.

Comment (sshah-wework) [inline build.gradle]: The [old version](https://plugins.gradle.org/plugin/com.diffplug.gradle.spotless/3.14.0) is still in gradle so this _should_ have succeeded.

Comment (sshah-wework) [reply build.gradle]: Opened up #90 to increase gradle's HTTP connection timeouts

Comment (wslulciuc) [reply build.gradle]: We don't really benefit from the spotless `3.15.0` release, closing this PR. Thanks!

Comment (sshah-wework) [reply build.gradle]: Gotcha, my fault, I thought this was a workaround to 3.14 failing to download.

Comment (wslulciuc) [reply build.gradle]: In a way, it was sort of a workaround and my attempt to secretly bump the version at the same time :)

**Pull request closed without merging**